sydent/http/httpclient.py

```diff
@@ -78,7 +78,7 @@ def post_json_get_nothing(self, uri, post_json, opts):
             b"Content-Type": [b"application/json"],
         }))
 
-        logger.debug("HTTP POST %s -> %s", json_str, uri)
+        logger.debug("HTTP POST %s", uri)
```

Comment:
I think that debug logs are fine because that log level shouldn't be set in production environments.

Reply:
Do we not run Sydent in debug mode right now?